blog.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: plantillablog
+title: Blog
+permalink: /blog
+---
+
+<p>Estás visitando el blog de Javier Polit!</p>
+
+<ol>
+    <li><a href="/blog/2024-03-08-post1">Post 1</a></li>
```

Comment:
Aquí no debes poner los posts así, sino con un listado automático:
https://jekyllrb.com/docs/posts/#displaying-an-index-of-posts

También debes preceder los enlaces con `{{site.baseurl}}` para que el enlace apunte al sitio correcto en GitHub Pages.